next.config.js

```diff
@@ -2,21 +2,15 @@
 /* eslint-disable no-param-reassign */
 
 const dotenv = require('dotenv');
-const withPlugins = require('next-compose-plugins');
-const nextImages = require('next-images');
-const nextFonts = require('next-fonts');
```

Comment:
was needed when Nextjs had no support for importing fonts without using plugins.

https://www.npmjs.com/package/next-fonts